Source/Core/DiscIO/WIABlob.cpp

```diff
@@ -1564,7 +1564,7 @@ WIARVZFileReader<RVZ>::ProcessAndCompress(CompressThreadState* state, CompressPa
 
     if (state->compressor)
     {
-      if (!state->compressor->Start())
+      if (!state->compressor->Start(entry.exception_lists.size() + entry.main_data.size()))
```

Comment:
I had to check that exceptions_list indeed stored bytes rather than exception structs (as the name would imply IMO). That's probably just because I'm unfamiliar with this code, though :)

Still, it might be worth renaming this member to exception_list_data for more clarity and for consistency with main_data in a followup PR?